jira_insight/insight.py

```diff
@@ -92,6 +98,21 @@ def __init__(self, insight, object_id, object_json=None):
             )
             self.attributes[attribute_object.name] = attribute_object
 
+    def update_object(self, attributes: dict):
+        attributes_json = []
+        for attribute_id, value in attributes.items():
+            entry = {
+                "objectTypeAttributeId": attribute_id,
+                "objectAttributeValues": [{"value": value}],
+            }
+            attributes_json.append(entry)
+        request_body = {"attributes": attributes_json}
```

Comment:
According to the [Docs](https://insight-javadoc.riada.io/insight-javadoc-8.6/insight-rest/#object__id__put), Insight requires the objectTypeId to be set when updating objects. So this needs to be added here, we could get it from `self.object_json` directly or we could extract the id in __init__.

Reply:
I believe this should do it, I chose the `self.object_json` route.

Reply:
Looks good, thank you!